layout/_third-party/analytics/index.swig

```diff
@@ -1,5 +1,6 @@
 {% include 'facebook-sdk.swig' %}
 {% include 'vkontakte-api.swig' %}
+{% include 'growingio.swig' %}
```

Comment:
Seems GA, Baidu and AI bigger services than growingio. So, only after that it must be included.